### DISCUSSION TITLE
On BEX build for v3 manifest, copy "_locale" folder

### DISCUSSION THREAD

**Discussion by ajaxe** [quasarframework/quasar#12928] [CLI - BEX mode] [answered]
Thanks for adding support for v3 manifest when using CLI with vite. I do notice the BEX build does not copy over "_locale" folder. "_locale" folder contains i18n json files required by chrome.i18n API.

Is there a way to extend the BEX build to include the "_locale" folder?

**Comment by rstoenescu** [comment]
Use the "assets" folder (which is copied "as-is") to store the locales too.

**Comment by ajaxe** [comment]
![image](https://user-images.githubusercontent.com/1263764/160309594-2cec9e5e-24fe-4e82-bb62-223071eb38b2.png)

I see the above error loading the unpackaged bex. Folder "_locales" is supposedly needs to be in the root of the extension. As documented in https://developer.chrome.com/docs/extensions/reference/i18n/#supported-locales.

**Reply by rstoenescu** [reply]
Ah. I see now. Will add support for this in the next q/app-vite version

**Reply by rstoenescu** [reply]
Please follow https://github.com/quasarframework/quasar/issues/12968 for the progress

**Comment by rstoenescu** [comment, marked as answer]
Fix will be available in q/app-vite v1.0.0-beta.10

**Reply by ajaxe** [reply]
thanks speedy response.
